tests/Integration/Behaviour/Features/Scenario/Discount/FO/full_user_experience_test_with_discount_cart_level.feature

```diff
@@ -1,6 +1,7 @@
 # ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s discount --tags full-ux-discount-test-cart-level
 @restore-all-tables-before-feature
 @full-ux-discount-test-cart-level
+@discount-fo
```

Comment:
This was for debugging I guess, not sure you need to keep them now that the behat tests are fixed